package.json

```diff
@@ -49,7 +49,9 @@
     "publishCmd": "npm publish",
     "publishDevCmd": "npm publish --tag ${devTag}",
     "versionCmd": "npm version --no-git-tag-version ${version}",
-    "cleanCmd": "git checkout -- package.json",
+    "checkFileTracking": "git ls-files --error-unmatch ${file}",
```

Comment:
You should have added all of these new defaults to the publishFixture definitions in the tests.  That was the cause of one failure.